contracts/rust/src/cape/events.rs

```diff
@@ -80,6 +80,7 @@ mod tests {
             &connection.contract,
             block_with_memos.clone(),
             BlockNumber::Latest,
+            10_000_000, // gas limit
```

Comment:
Why do we have a magic number here? Should not it be `DEFAULT_RELAYER_GAS_LIMIT` instead?

Reply:
Yes but that one is defined in the relayer crate.